v4/export/dump.go

```diff
@@ -775,6 +776,9 @@ func tidbStartGCSavepointUpdateService(d *Dumper) error {
 func updateServiceSafePoint(ctx context.Context, pdClient pd.Client, ttl int64, snapshotTS uint64) {
 	updateInterval := time.Duration(ttl/2) * time.Second
 	tick := time.NewTicker(updateInterval)
+	rand.Seed(time.Now().UnixNano())
+	dumplingServiceSafePointID := fmt.Sprintf("%s_%04d", dumplingServiceSafePointPrefix, rand.Int()%10000)
```

Comment:
maybe just use `time.Now().UnixNano()` directly? a 4-digit number has 1% collision rate.

```suggestion
	dumplingServiceSafePointID := fmt.Sprintf("%s_%d", dumplingServiceSafePointPrefix, time.Now().UnixNano())
```